File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist6.cs
// <snippetdistributecampaignfrommarketinglist6>


 // Add a list of contacts to the marketing list.
 var addMemberListReq = new AddListMembersListRequest
 {
     MemberIds = new[] { _contactIdList[0], _contactIdList[2] },
     ListId = _marketingListId
 };

 _serviceProxy.Execute(addMemberListReq);

 Console.WriteLine("  Contacts with GUIDs \r\n\t{{{0}}}\r\n\tand {{{1}}}\r\n  were added to the list.",
     _contactIdList[0], _contactIdList[1]);

// </snippetdistributecampaignfrommarketinglist6>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist6.vb
' <snippetdistributecampaignfrommarketinglist6>


	' Add a list of contacts to the marketing list.
          Dim addMemberListReq =
              New AddListMembersListRequest With
              {
                  .MemberIds = {_contactIdList(0), _contactIdList(2)},
                  .ListId = _marketingListId
              }

	_serviceProxy.Execute(addMemberListReq)

          Console.WriteLine("  Contacts with GUIDs " _
                            &amp; vbCrLf &amp; vbTab &amp; "{{{0}}}" _
                            &amp; vbCrLf &amp; vbTab &amp; "and {{{1}}}" _
                            &amp; vbCrLf &amp; "  were added to the list.",
                            _contactIdList(0),
                            _contactIdList(1))

' </snippetdistributecampaignfrommarketinglist6>